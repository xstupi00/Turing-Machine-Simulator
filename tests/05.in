S a B c
B c A d
A d A R
A   C R
C   F  
a
