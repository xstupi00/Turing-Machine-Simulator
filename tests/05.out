Sa
Bc
Ad
dA 
d C 
d F 
